### DISCUSSION TITLE
Quem somos nós

### DISCUSSION THREAD

**Discussion by vitorestevam** [clube-de-leitura-python/pense-em-python-2021#6] [Conversa geral]
# Quem somos nós?

> Esse é um espaço pra gente se apresentar 👋😉

No encontro foi sugerido que falássemos nome, área de interesse e uma rede para contato.

**Comment by vitorestevam** [comment]
Sou Vitor Estevam e estou participando do grupo pra me motivar a alavancar meus estudos com Python, visando "solidificar" o que eu sei da linguagem. 👨‍💻

[@vitorstvm](https://twitter.com/vitorstvm) no twitter

**Comment by CatarinaLuciane** [comment]
Meu nome é Catarina e estou participando pra conversar sobre meus aprendizados e, principalmente, encontrar gente mais experiente em python. Na verdade, sou professora de português, mas me interesso em estar sempre aprendendo algo novo. Dessa vez, me propus a entender um pouco dessa linguagem pra estimular o raciocínio lógico. Li também que pode ser bacana pra desenvolver mais ''resolução de problemas'', então estou levando como um xadrez virtual.

Não sou nada boa com matemática & assuntos relacionados, evidentemente kkkk mas estou tentando retomar o contato com os números.

@clucyzinha no twitter

**Comment by villares** [comment]
É legal todo mundo se apresentar aqui neste thread!
Sou Alexandre Villares e eu [tento fazer um desenho por dia com Python](https://github.com/villares/sketch-a-day), eu devia gastar menos tempo no Twitter: [@villares](https://twitter.com/villares).

**Comment by rafaotetra** [comment]
Meu nome é Rafael, sou tetraplégico, atuo como engenheiro DevOps no Dell Lead.

Estou buscando aprender mais a linguagem Python para automatizar as minhas tarefas repetitivas e conseguir melhorar minha produtividade.

@rafaotetra no Twitter e Telegram

**Comment by omyasuda** [comment]
Meu nome é Osvaldo Yasuda sou aposentado e estou a disposição para lermos juntos o **Pense em Python** (2ª ed.), de **Allen B. Downey**!

Vamos lá? Isso pode ser divertido e bastante útil no nosso dia-a-dia!

Estou a disposição 

> Aqui em nossa sala de discussões

> no telegram https://t.me/omyasuda

> ou @omyasuda em nas Redes Sociais

**Comment by rgobatto** [comment]
Sou Rodrigo, arquiteto e entusiasta das tecnologias digitais. 

Estou em busca de aprender mais sobre da programação e seus fundamentos e aplicar na minha vida de forma geral. Tenho interesse em aplicar na arquitetura, principalmente no que diz respeito a design paramétrico, e desenvolver projetos pessoais, como um aplicativo pra marcar ponto de tranca pra aposentar de vez o papel e a caneta, haha.

@rgobatto no instagram

**Comment by gbrlbrt** [comment]
Olá, meu nome é Gabriel. Sou pesquisador e estou preparando meu doutorado em direito.

Não tenho nenhuma conhecimento de programação, mas quero aprender a programar para aplicar nas minhas pesquisas.

Abs

**Comment by albanesere** [comment]
Sou Regina Albanese tenho 57 anos, sou estatística há 7 anos, e antes disse fui professora na educação básica e psicopedagoga. Desde 2009 sou professora unviersitária. Estou migrando de soft proprietário para soft livres na minha área. Comecei com o R, e parei... resolvi parender python pois me falaram que parecia fortran e cobol... e eu fui programadora quando "xóffen" desde 1983 até 1984... dai resolveram que eu deveria ser professora... porque eu casei.... e eu aceitei... agora resolvi revisar meus desejos interrompidos. Comecei aqui e alguns percalços atingiram meu percurso... mas... já estou em linha com vcs. Quero me desenvolver  mais...

**Reply by albanesere** [reply]
pois... eu tenho um autismo leve... e as vezes eu não compreendo algumas coisas e as vezes eu também troco letras (como fiz aqui no comentário, desculpem... deveria ter lido antes...)

**Reply by albanesere** [reply]
acabei de ler as pautas das reuniões e eu vou deixar aqui meu interesse
quero desenvolver aulas de estatistica usando o pyhton e fazer as analises de minha consultoria.
tenho feito alguns projetos, mas ainda tem muito a aprender, logo, não me interessa, à priori, a parte da computação, mas, já vi que devo aprender, pois em muitos casos, posso automatizar muitas coisas....

**Comment by claudewick** [comment]
Sou o Claude, tenho 45 anos e sou da área jurídica. Eu não consigo ficar parado e como estava enjoado de estudar assuntos da minha área, resolvi fazer faculdade de Sistemas de Informação(estou no 3º ano). Estou particularmente interessado em segurança da informação e preservação de informações digitais. Tive algum contato com diversas linguagens na faculdade, mas nenhuma me interessou o suficiente e desde o ano passado estou estudando Python por conta.
